messaging/kafka-avro-reactive-messaging/src/main/resources/docker-compose-strimzi.yaml

```diff
@@ -0,0 +1,44 @@
+version: '2'
```

Comment:
Maybe, better to move this file into `src/main/resources/docker` for related docker stuff as the rest of the modules.